src/lib/coda_networking/coda_networking.ml

```diff
@@ -546,8 +546,11 @@ module Make (Inputs : Inputs_intf) = struct
                     Staged_ledger_hash %{sexp: Staged_ledger_hash.t}"
                   peer staged_ledger_hash ;
                 Ok
-                  (Envelope.Incoming.wrap ~data:staged_ledger_aux
-                     ~sender:(fst peer)) )
+                  (Envelope.Incoming.wrap
+                     ~data:
+                       staged_ledger_aux
+                       (* TODO: this isn't really a discovery port, how to handle? *)
```

Comment:
As in, the envelope doesn't want a discovery port, or the peer doesn't have a real discovery port? The peer should be fine I think? I think the port maybe shouldn't even be in the envelope, but that's work for another day... Does anything need to be done with this TODO to land this PR?

Reply:
The sender doesn't have a real discovery port. Eventually,  the goal should be to have `sender` in envelopes be a `Peer.t` (as in #1367), but that's beyond this PR.

Nothing needs to be done for the TODO in this PR.